### DISCUSSION TITLE
nav/

### DISCUSSION THREAD

**Discussion by giscus** [Yiov/vitepress-doc#6] [General]
# nav/

我的vitpress文档教程

https://vitepress.yiov.top/nav/

**Comment by h7ml** [comment]
茂神牛批

**Comment by Seasir-Hyde** [comment]
https://yiov.top/guide/daily.html
大佬这个导航网站源码有吗，求一份~谢谢了~

**Reply by Yiov** [reply]
导航是教程茂神的，我就修改了下样式。标签导航用ai写的，bug挺多的，也整不来

**Comment by Moeus** [comment]


**Reply by Yiov** [reply]
对react不太了解，或许使用vite-plugin-react插件能实现

**Comment by 253715** [comment]
import MNavLink from './MNavLink.vue'  这句里面MNaVLink直接爆红了是为什么

**Reply by Yiov** [reply]
缺少MNavLink.vue这个组件，查看自己目录是否存在这个文件，或者文件内容为空

**Comment by sendudesign** [comment]
https://sendudesign.com/nav/
请问右上角导航没有变色效果是哪里的问题？

**Reply by Yiov** [reply]
哪个右上角？是单个导航标签写 `茂神牛批` 的那个吗？在 `data.ts` 数据库中配置badge参数即可

**Reply by sendudesign** [reply]
大佬请点开我的网站，点击右上角的导航不会变色（选中效果），点击首页和帮助可以。

**Reply by Yiov** [reply]
那肯定是 `config.ts` 中nav配置有问题，也许你路径掉符号也有可能。

```diff
- { text: '导航', link: 'nav/' },
+ { text: '导航', link: '/nav/' },
```

**Reply by sendudesign** [reply]
解决~谢谢大佬
